### PR TITLE
Update ProphetStor Federator.ai

### PR DESCRIPTION
**Pull request by chenyingtz**
Update ProphetStor Federator.ai integration
- Update ProphetStor logo
- Fix Federator.ai dashboard issues.

### What does this PR do?

Update ProphetStor Federator.ai integration for changing ProphetStor logo and fixing Federator.ai dashboard issues.

### Motivation

Update ProphetStor Federator.ai integration for changing ProphetStor logo and fixing Federator.ai dashboard issues.

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Comment by hithwen** [comment]
Hallo, I've requested the logos to be updated internally

**Comment by chenyingtz** [comment]
> Hallo, I've requested the logos to be updated internally

Thanks!

File: federatorai/manifest.json [hunk 1]
```diff
@@ -29,7 +29,6 @@
 	},
     "saved_views": {},
     "monitors": {
-		"Recommended Replica Number Reaches Max Limit": "assets/recommended_monitors/federatorai_recommended_replica_num.json",
```
Comment (coignetp) [inline federatorai/manifest.json:32]: Is it expected to remove the monitor?

**Review by coignetp — APPROVED** [review]
Hi, the dashboards look good on staging, so I approved, but I wanted to check if deleting a recommended monitor is expected here

File: federatorai/manifest.json (hunk 1 above)
Comment (chenyingtz) [reply federatorai/manifest.json:32]: yes, we decide to remove this monitor.

Comment (coignetp) [reply federatorai/manifest.json:32]: Thanks for your answer, I'm merging it then!

**Pull request merged**